src/hooks/useTxHistory.ts

```diff
@@ -29,7 +30,7 @@ const useTxHistory = (
 
       return filter
         ? fetchFilteredTxHistory(chainId, safeAddress, filter, pageUrl)
-        : getTransactionHistory(chainId, safeAddress, pageUrl)
```

Comment:
Wouldn't this call only fetch trusted transactions but now with the change would also fetch untrusted ones?

Reply:
Yes, and this is correct. That it wasn't passed was incorrect. We want to always load untrusted txs. This bug didn't manifest itself only because gateway apparently ignores the query when a pageUrl is passed.

Reply:
Wow, nice catch then! I was already wondering how we currently display untrusted transactions in the history even though we don't pass false here. 💯